unitTests/MockEnvironment.ts

```diff
@@ -95,7 +95,7 @@ export class MockEnvironmentBuilder {
     if (this.built) {
       return this.getBindings();
     }
-    if (this.element.parentNode == undefined) {
+    if (!this.element.parentNode) {
```

Comment:
I don't know if this or `Utils.isNullOrUndefined` would be cleaner, but since `parentNode` is never going to be 0, false or an empty string, all three solutions (including a `==`) are equivalent anyways.